### ISSUE TITLE
gns3-server: need to remove to fix conflicts

### ISSUE THREAD

**Issue by CameronNemo** [void-linux/void-packages#10774]
### System

* xuname:  Void 5.0.5_1 x86_64 GenuineIntel notuptodate rrdFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
* package:  gns3-server-2.1.15_2

### Expected behavior

Updates install cleanly.

### Actual behavior

```
CONFLICT: python3-prompt_toolkit1-1.0.15_1 with python3-prompt_toolkit-2.0.9_1 in transaction (matched by python3-prompt_toolkit>=2)
CONFLICT: python3-prompt_toolkit-2.0.9_1 with python3-prompt_toolkit1-1.0.15_1 in transaction (matched by python3-prompt_toolkit1)
```

### Steps to reproduce the behavior

Upgrade from python3-prompt_toolkit 1.0.15_2 to 2.0.9_1 with gns3 installed.

**Comment by CameronNemo** [comment]
I was able to install gns3-server just fine.

We really need to take these transitions step by step until we better understand the process they call for.

**Comment by jnbr** [comment]
This is not fully solved yet. Reverting python3-prompt_toolkit didn't work:
```
xbps-query -R -p pkgver python3-prompt_toolkit
python3-prompt_toolkit-2.0.9_1
```

**Comment by jnbr** [comment]
should be fixed with #10903 now.

```
xbps-query -R -p pkgver python3-prompt_toolkit
python3-prompt_toolkit-1.0.15_3
```
